src/ui/components/Icon/styles.scss

```diff
@@ -115,3 +115,7 @@
 .Icon-featured {
   @include icon($name: "featured");
 }
+
+.Icon-restart-required {
```

Comment:
I think the name should match the icon file/etc. So this should just be called `.Icon-restart`.

Reply:
Ah, yes, renaming this to `.Icon-restart` would have cleared up a lot of the confusion I had initially